contracts/contracts/GrantRoundManager.sol

```diff
@@ -69,39 +70,32 @@ contract GrantRoundManager is SwapRouter {
   }
 
   // --- Core methods ---
-
   /**
    * @notice Creates a new GrantRound
    * @param _owner Grant round owner that has permission to update the metadata pointer
    * @param _payoutAdmin Grant round administrator that has permission to payout the matching pool
    * @param _matchingToken Address for the token used to payout match amounts at the end of a round
-   * @param _registry Address that contains the grant metadata
    * @param _startTime Unix timestamp of the start of the round
    * @param _endTime Unix timestamp of the end of the round
    * @param _metaPtr URL pointing to the grant round metadata
-   * @param _minContribution Miniumum donation amount that can be made using the given donation token
    */
   function createGrantRound(
     address _owner,
     address _payoutAdmin,
     IERC20 _matchingToken,
-    GrantRegistry _registry,
     uint256 _startTime,
     uint256 _endTime,
-    string memory _metaPtr,
-    uint256 _minContribution
+    string memory _metaPtr
   ) external {
     require(_matchingToken.totalSupply() > 0, "GrantRoundManager: Invalid matching token");
     GrantRound _grantRound = new GrantRound(
       _owner,
       _payoutAdmin,
-      _registry,
```

Comment:
Rather than removing this parameter from the `GrantRound` constructor, I think we want to remove it from the `createGrantRound` method parameters, and instead pass the storage variable `registry` in the `GrantRoundManager` through to the `GrantRound`. The Round should have some publicly accessible hold on the registry it refers to, otherwise the `grantRoundIds` it refers to in its metadata would be meaningless.

Reply:
Hmm, why does the registry need to be part of the `GrantRound` on chain? It can't hurt, but just want to make sure we're on the same page. Some thoughts:
- If the `grantRoundIds` are defined off-chain in the metadata, you can put the registry there also, and if we want to make verifying that easier we should index the grant round address in the `GrantRoundCreated` event (maybe we should index that anyway)
- We currently have an on-chain link (in the form of events) to go from GrantRoundManager to the GrantRounds it's created
- The GrantRound contract does not need to know anything about the GrantRoundManager on-chain as of now

Reply:
Decision from today's sync was that we'll keep the registry parameter here as described in @apbendi's original comment. Reimplemented in 41e22348c24784cd2af68404b9b8fe4e9a87f030. Added ~11k gas to the `createGrantRound` cost for reference